.gitignore

```diff
@@ -162,3 +162,6 @@ dictionary.dic
 
 # Man pages
 man
+
+generated
+test_box
```

Comment:
Why do we need to add this?

Reply:
I have the folders locally to make it easy to test. So I just checked in. I think it doesn't hurt.

Reply:
You shouldn't be doing this on the upstream .gitignore, just use your global gitignore file on your system. See this [doc](https://git-scm.com/docs/gitignore). That part:

> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.